protected/controllers/PracticeController.php

```diff
@@ -74,9 +74,10 @@ public function actionCreate($context = null)
                 $context === 'AJAX');
         }
         if ($context === 'AJAX') {
```

Comment:
I thought the AJAX context thing was fixed ages ago...

Reply:
I think we said we'd revisit it in Phase 2.